### DISCUSSION TITLE
How to change shortcut in OSX?

### DISCUSSION THREAD

**Discussion by dhimanda** [Jatana/FastOlympicCoding#35] [Q&A] [answered]
I know there is a shortcut ctrl+b to build (OSX), 
I want to change it into ctrl + option + b, 
How to do this?

**Comment by Jatana** [comment, marked as answer]
Change these two lines in the file `Default (OSX).sublime-keymap` in your local `FastOlympicCoding` folder to `ctrl+alt+b`:
https://github.com/Jatana/FastOlympicCoding/blob/master/Default%20(OSX).sublime-keymap#L235
https://github.com/Jatana/FastOlympicCoding/blob/master/Default%20(OSX).sublime-keymap#L253

**Reply by dhimanda** [reply]
WOW! You are so cool, man!  Take love bro <3.
